source/index.html.md

```diff
@@ -651,15 +651,60 @@ This endpoint schedule payments for this subscription using the dates and amount
 
 This endpoint schedule payments for this subscription using the order and amounts specified in the body
 
-> **Example request**
+> **Example body**
 
 ```json
+[
   {
-    "amount": 27,
+    "amount": 20,
     "order": 0
+  },
+  {
+    "amount": 40,
+    "order": 1
+  },
+  {
+    "amount": 60,
+    "order": 2
   }
+]
+```
+
+> **Example response for a schedule payment of a subscription using the order of payments**
+
+```json
+
+{
+  "customerId": "91c2ebca-a516-11ec-b909-0242ac120002",
+  "subscriptions": 
+    {
+      "id": "e537205a-7bd1-45ac-b709-0242ac1125879",
+      "amount": 10,
+      "nextBillingDate": "2022-03-30T10:30:18Z",
+      "schedules":
+        [
+          {
+            "amount": 20,
+            "order" : 0
```

Comment:
The response will contain date instead of order. So use `date` and then provide use the same dates as in the previous example.